.ci_support/migrations/python314.yaml

```
@@ -1,26 +1,24 @@
+# this is intentionally sorted before the 3.13t migrator, because that determines
+# the order of application of the migrators; otherwise we'd have to add values for
+# is_freethreading and is_abi3 keys here, since that migration extends the zip
 migrator_ts: 1724712607
```

This now needs to match the values from https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/7690
```suggestion
migrator_ts: 1724712514
```